workflows/isoseq.nf

```diff
@@ -93,8 +185,9 @@ workflow ISOSEQ {
     ch_multiqc_files = ch_multiqc_files.mix(Channel.from(ch_multiqc_config))
     ch_multiqc_files = ch_multiqc_files.mix(ch_multiqc_custom_config.collect().ifEmpty([]))
     ch_multiqc_files = ch_multiqc_files.mix(ch_workflow_summary.collectFile(name: 'workflow_summary_mqc.yaml'))
-    ch_multiqc_files = ch_multiqc_files.mix(CUSTOM_DUMPSOFTWAREVERSIONS.out.mqc_yml.collect())
```

Comment:
I saw you are not using the `CUSTOM_DUMPSOFTWAREVERSIONS` module. This module takes all the versions from the different tools and them to the multiqc report, so you might need to include it back.

Reply:
`CUSTOM_DUMPSOFTWAREVERSIONS` is in again